src/types/venues.ts

```diff
@@ -145,6 +145,7 @@ export interface BaseVenue {
   miniAvatars?: boolean;
   adultContent?: boolean;
   samlAuthProviderId?: string;
+  samlProfileMappings?: Mappings;
```

Comment:
As called out elsewhere, i'm not sure that we 'gain anything' by using `Mappings` here, vs just using the underlying types directly.

Reply:
And if we are going to use it, it would seem that the type should live in this file, not `User`. And probably have a more specific name to communicate its purpose, eg. `SAMLProfileMappings` or similar.

Reply:
Done in https://github.com/sparkletown/sparkle/pull/1667/commits/2d4f2617700b9bb64a4edaab5c88b7bcb223540c